login_server/datatypes.py

```diff
@@ -2454,6 +2454,7 @@ def set_player(self, unique_id, name, tag, loadouts):
         self.findbytype(m06de).set(tag)
 
         loadout_arrays = []
+        loadout_overall_idx = 0
```

Comment:
Does it work also when you have it start at 1 and use its value without adding 0x2730? I think 0x2730 is just an arbitrary offset, so a normal index would be more clean.

Reply:
Done. Works fine with just an index (even starting at 0).